src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/GSuiteSteps.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.ons.fsdr.tests.acceptance.steps;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cucumber.api.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import uk.gov.ons.fsdr.tests.acceptance.utils.GsuiteMockUtils;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+@Slf4j
+@PropertySource("classpath:application.properties")
+public class GSuiteSteps {
+
+  @Autowired
+  private GsuiteMockUtils gsuiteMockUtils;
+
+  @Value("${service.rabbit.url}")
+  private String rabbitLocation;
+
+  @Value("${service.rabbit.username}")
+  private String rabbitUsername;
+
+  @Value("${service.rabbit.password}")
+  private String rabbitPassword;
+
+  private final ObjectMapper objectMapper = new ObjectMapper();
+
+  @Then("the employee is correctly created in gsuite with roleId {string} and orgUnit {string}")
+  public void the_employee_is_correctly_updated_in_gsuite(String roleId, String orgUnit) {
+    String[] records = gsuiteMockUtils.getRecords();
+
+    assertThat(records[0]).contains(
+        "\"changePasswordAtNextLogin\":true,\"customSchemas\":{\"Employee_Information\":{\"RoleID\":\"" + roleId
```

Comment:
comparing json as strings makes the tests brittle as order of fields and whitespace are involved in the comparison,, better way is to have the as  json objects (you could read the strings) , and then do the compare on the jsomn object.
https://collaborate2.ons.gov.uk/jira/browse/FWMT-1875
Dont fix it in this code review Ive created this ticket.